src/theme.rs

```diff
@@ -32,17 +32,17 @@ pub mod colors {
     pub const LIGHT_GREY: Color = Color::grey8(0xe7);
     pub const MEDIUM_GREY: Color = Color::grey8(0x88);
     pub const SIDEBAR_EDGE: Color = Color::grey8(0xc7);
-    pub const HIGHLIGHT_COLOR: Color = Color::rgb8(0xA6, 0xCC, 0xFF);
+    pub const HIGHLIGHT_COLOR: Color = Color::rgb8(0xFF, 0xEE, 0x55);
```

Comment:
something to keep in mind is that this is also the color for selected text; if we want those to be different we could add a new constant, and use it *just* for the selected glyph background color.

Reply:
I force pushed a new update (4d781ac) that fixes this. I just used some temporary `Color::rgb8` colors for now. I didn't want to add any new constants since that will hopefully get worked out in a separate PR improving the theming system at some point. Please feel free to make any needed changes before merging.